verification/curator-service/ui/src/components/CaseForm.tsx

```diff
@@ -210,6 +210,8 @@ const initialValuesFromCase = (
         },
         events: {
             ...c.events,
+            dateEntry: toLocalDate(c.events.dateEntry),
+            dateReported: toLocalDate(c.events.dateReported),
```

Comment:
Is there any reason not to report in UTC everywhere? I can see the value of timezones in data collection, less so for reporting.